docs/lists_degrees.md

```diff
@@ -122,10 +122,60 @@ Quality: Manually updated on an ad-hoc basis. Please submit a pull request if in
 |---|---|---|
 | `id` | UUID | A unique identifier. The same as `dttp_id` if that field is non-`nil`, otherwise a new UUID was minted at import time. |
 | `name` | string | The long name of the subject, eg "accountancy" |
-| `synonyms` | string array | A list of common alternative names |
+| `suggestion_synonyms` | string array | A list of common alternative names that *may* be appropriate for this subject |
+| `match_synonyms` | string array | A list of common alternative names that are equivalent to this subject. An answer matching a match synonym can be safely matched to this subject.|
 | `dttp_id` | uuid | The ID used for this subject in DTTP |
 | `hesa_itt_code` | string | The ID used for this subject in HESA |
 
+### `DfE::ReferenceData::Degrees::COMMON_COMPOUND_SUBJECTS`
```

Comment:
'Compound' feels like a jargony word to me that I wouldn't use naturally to describe these subjects... Maybe 'combined' or 'joint'?  Would also suggest dropping 'common' as we don't actually have a ton of data to support this other than applications to initial teacher training, and perhaps in future we might even add some less common ones (or existing common ones will become less common).

We could still say in a note that we generally only add combined subjects commonly seen, and don't intend the list to be exhaustive

```suggestion
### `DfE::ReferenceData::Degrees::COMBINED_SUBJECTS`
```